pkg/generate/app/cmd/newapp.go

```diff
@@ -218,6 +218,11 @@ func (c *AppConfig) SetOpenShiftClient(osclient client.Interface, originNamespac
 		ClientMapper: c.clientMapper,
 		Namespace:    originNamespace,
 	}
+	c.dockerSearcher = app.ImageImportSearcher{
```

Comment:
@smarterclayton if a local docker connection is available, this would be overwritten by https://github.com/openshift/origin/pull/6288/files#diff-67f42376bdba60c035c6268d160059baR188
Is that intended?